src/extension/ios/plistBuddy.ts

```diff
@@ -37,7 +37,9 @@ export class PlistBuddy {
             } else {
                 productsFolder = path.join(iosProjectRoot, "build", "Build", "Products");
             }
-            const sdkType = simulator ? "iphonesimulator" : "iphoneos";
+            const platform = (scheme && scheme.toLowerCase().includes("tvos")) ? "appletv" : "iphone";
+            const device = simulator ? "simulator" : "os";
+            const sdkType = `${platform}${device}`;
```

Comment:
I suppose that it is better to store strings like `appletv` and `iphone` as an array or object of constants and then use them in here.

Reply:
Also the implementation of `sdkType` selection algorithm could be moved into a separate function.